### ISSUE TITLE
Serialize to Stream?

### ISSUE THREAD

**Issue by stevenranney** [microsoft/fsharplu#9]
Can I serialize with `Microsoft.FSharpLu.Json` to a stream?  I've asked [this question on StackOverflow](http://stackoverflow.com/questions/41983955/using-microsoft-fsharplu-to-serialize-json-to-a-stream).

Right now, in `Compact`, I see a number of methods, some that can deserialize from a stream, but nothing that allows me to serialize to a stream.  Is it possible to do with `Compact`?

**Comment by blumu** [comment]
The helper is missing from `Compact` but you should be able to do it by following the C# example from 
http://www.newtonsoft.com/json/help/html/SerializingJSON.htm. Something along the lines:
```
let product = new Product()
let serializer = new JsonSerializer()
serializer.Converters.Add(new Microsoft.FSharpLu.Json.CompactUnionJsonConverter())
use sw = new StreamWriter(@"c:\json.txt") /// replace by your own stream writer
use writer = new JsonTextWriter(sw)
serializer.Serialize(writer, product)
```

**Comment by stevenranney** [comment]
PERFECT!  Thanks so much for your help.

**Comment by blumu** [comment]
Reopening, will be addressed by PR #67

**Comment by blumu** [comment]
Implemented in #67
